### DISCUSSION TITLE
Implementation in NAMD

### DISCUSSION THREAD

**Discussion by HanatoK** [SSAGESLabs/PySAGES#315] [General]
Hi! I am interested in making PySAGES working with NAMD. I read the `DLExt.h` in the openMM and LAMMPS implementations but am still a bit perplexed. Is there any documentation about how to interface with new MD engines? What do I need to implement from the NAMD side?

**Comment by p-zubieta** [comment]
Hi @HanatoK. Sorry for the slow response here. This probably requires discussing the details over a video call. If you're interested we can setup one. I can try to summarize things here anyway. Give me a couple of days to come up with a more detailed description on how our plugins work.
